internal/dvcore/run.go

```diff
@@ -198,6 +200,11 @@ func pullBatches(targets []multipmuri.Entity, h *cayley.Handle, githubToken stri
 
 				githubprovider.FetchRepo(ctx, repo, githubToken, out, ghOpts)
 			}(target)
+			case multipmuri.TrelloProvider:
+				go func(board multipmuri.Entity) {
+					defer wg.Done()
+					trelloprovider.FetchCard(ctx, board, trelloToken, trelloApiKey, target.LocalID()[1:], out)
```

Comment:
Btw, from `pullBatches`, I expect my drivers to fetch not only `Trello Card` but also `Trello Board`.

The helper here should be named `trelloprovider.FetchTarget` or something more flexible I guess.